bundles/persistence/org.openhab.persistence.dynamodb/META-INF/MANIFEST.MF

```diff
@@ -20,20 +20,20 @@ Import-Package: com.google.common.collect;version="10.0.1",
  org.osgi.service.event,
  org.slf4j
 Bundle-ClassPath: .,
-  lib/aws-java-sdk-core-1.11.56.jar,
-  lib/aws-java-sdk-dynamodb-1.11.56.jar,
-  lib/aws-java-sdk-kms-1.11.56.jar,
-  lib/aws-java-sdk-s3-1.11.56.jar,
+  lib/aws-java-sdk-core-1.11.213.jar,
+  lib/aws-java-sdk-dynamodb-1.11.213.jar,
+  lib/aws-java-sdk-kms-1.11.213.jar,
+  lib/aws-java-sdk-s3-1.11.213.jar,
   lib/commons-codec-1.9.jar,
   lib/commons-logging-1.1.3.jar,
   lib/httpclient-4.5.2.jar,
   lib/httpcore-4.4.4.jar,
-  lib/ion-java-1.0.1.jar,
+  lib/ion-java-1.0.2.jar,
   lib/jackson-annotations-2.6.0.jar,
-  lib/jackson-core-2.6.6.jar,
-  lib/jackson-databind-2.6.6.jar,
-  lib/jackson-dataformat-cbor-2.6.6.jar,
-  lib/jmespath-java-1.11.56.jar,
+  lib/jackson-core-2.6.7.jar,
+  lib/jackson-databind-2.6.7.1.jar,
+  lib/jackson-dataformat-cbor-2.6.7.jar,
+  lib/jmespath-java-1.11.213.jar,
   lib/joda-time-2.8.1.jar
 Service-Component: OSGI-INF/dynamodb.xml
 Bundle-Activator: org.openhab.persistence.dynamodb.internal.DynamoDBPersistenceServiceActivator
```

Comment:
Can you insert a `Bundle-ActivationPolicy: lazy` here?

Reply:
I can surely do this but did not find this from any other persistence bundles in openHAB1-addons: dynamodb would be the first one. Please note that the component is `immediate` (similar to other persistence bundles). Does it matter what is the `Bundle-ActivationPolicy` with `immediate` components since the component is instantiated immediately?

Reply:
I have little knowledge of what this field does. I was just going from the Eclipse warning that states:

````
Bundles with a Service-Component should set the Bundle-ActivationPolicy to lazy.
````

I'm always annoyed with warnings that are issued for no reason.

Since none of the other persistence models have this, I'm OK with omitting it here too, but I'd really like to understand why this warning is so pervasive.

Reply:
Makes sense to remove warnings. Added a commit.